flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64", "arm", "aarch64"]
+}
```

Comment:
```suggestion
{
  "publish-delay-hours": 1
}
```

Reply:
Why should `"only-arches"` be removed?

Reply:
Only x86_64 and aarch64 are supported these days. I don't understand the suggestion to use `publish-delay-hours` though, please just remove this file completely.